app/services/importers/create_npq_contract.rb

```diff
@@ -4,14 +4,16 @@
 
 module Importers
   class CreateNPQContract
-    attr_reader :path_to_csv
+    attr_reader :path_to_csv, :new_course_flag
 
-    def initialize(path_to_csv:)
+    def initialize(path_to_csv:, new_course_flag: false)
       @path_to_csv = path_to_csv
+      @new_course_flag = new_course_flag
     end
 
     def call
       check_headers
+      version = Finance::Statement::NPQ.latest.contract_version if new_course_flag == true && Finance::Statement::NPQ.count.positive?
```

Comment:
I think to get the latest version for a contract we'll prob need the following logic:
```
Finance::Statement::NPQ.where(cohort: , cpd_lead_provider:).pluck(:contract_version).sort.last
```
so might be best moving this into the row block as might be different between providers.
Also do you mind adding a spec for this as well?